Configuration/AlCa/python/autoCond.py

```diff
@@ -20,9 +20,11 @@
     # GlobalTag for MC production (Heavy Ions collisions) with optimistic alignment and calibrations for Run2
     'run2_mc_hi'        :   '80X_mcRun2_HeavyIon_v3',
     # GlobalTag for Run1 data reprocessing
-    'run1_data'         :   '80X_dataRun2_v4',
+    'run1_data'         :   '80X_dataRun2_v5',
```

Comment:
i'll remove the run1 data change sorry that was a test, but the second one is there to remove the conflict..